examples/fl-tabular/task.py

```diff
@@ -0,0 +1,126 @@
+import torch.nn as nn
+from flwr_datasets import FederatedDataset
+from sklearn.preprocessing import OneHotEncoder
+import pandas as pd
+import torch
+from torch.utils.data import TensorDataset, DataLoader
+import torch.optim as optim
+from collections import OrderedDict
+
+NUMBER_OF_CLIENTS = 5
+
+
+def load_data(partition_id: int):
+    fds = FederatedDataset(
+        dataset="scikit-learn/adult-census-income",
+        partitioners={"train": NUMBER_OF_CLIENTS},
+    )
```

Comment:
wouldn't it be better to remove this out of this function ? in theory creating the federated dataset is only needed once, leaving `load_data` to simply load a particular client partition.